src/main.go

```diff
@@ -5,22 +5,18 @@ import (
 	"os"
 	"strconv"
 
+	"github.com/Real-Dev-Squad/gopher-cloud-service/src/config"
 	"github.com/Real-Dev-Squad/gopher-cloud-service/src/routes"
 )
 
 func main() {
-	value, keyExists := os.LookupEnv("env")
-	port := strconv.Itoa(8000)
-	version := strconv.Itoa(1)
 
-	if keyExists {
-		log.Printf("Running in %v mode\n", value)
-	} else {
-		log.Printf("Running in local dev mode\n")
-		value = "local"
-	}
+	env, _ := os.LookupEnv("env")
 
-	router := routes.SetupRouter(value, version)
-	log.Printf("Server is running on http://localhost:%v\n", port)
+	config.Setup(env)
+	port := strconv.Itoa(config.Global.Port)

```

Comment:
Why are we converting it to a string here?

Reply:
So the Port number is being provided as the number in toml files.
But here: https://github.com/Real-Dev-Squad/website-learn-backend/pull/10/files/8d3aa05f20afee0693f905c091d801564e08cfab#diff-9e185f29fa355d7dd8fdd9c9ff1d0723b85206aa7d37c4eec93997005dc291ebR21

we are using string concatenation to run the router